### ISSUE TITLE
How to use this model to evaluate other GAN model 

### ISSUE THREAD

**Issue by jononny** [hanzhanggit/StackGAN-inception-model#3]
I used  >python inception_score.py --image C:\Users\StackGAN-inception-model-master\image

and got the error like:

InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [2048,51] rhs shape= [2048,21]
         [[Node: save/Assign_31 = Assign[T=DT_FLOAT, _class=["loc:@logits/logits/weights"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](logits/logits/weights, save/RestoreV2:31)]]
         [[Node: save/RestoreV2/_59 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_84_save/RestoreV2", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

How could I solve this? please help me! 
Well, I see some of the

**Comment by jononny** [comment]
Well, I solve this by using the parameter 'num_classes' = 20
and Also using 128 images in an image folder, can this program run correctly.
use like: python inception_score.py --image_folder C:\Users\StackGAN-inception-model-master\image
